Api/ProductLabelServiceInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
This interface is not used, probably it can be deleted :)

Reply:
Done! i forgot it